pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/VedtakEndringVedFlyttingMellomLand.kt

```diff
@@ -0,0 +1,726 @@
+                            )
+                        ) and
+                        (eksportForbudKode.equalTo(FLYKT_ALDER) or eksportForbudKode.equalTo(UFOR25_ALDER)
+                                or eksportForbudKodeAvdoed.equalTo(FLYKT_ALDER) or eksportForbudKodeAvdoed.equalTo(
+                            EksportForbudKode.DOD26_ALDER
+                        ))
+            ) {
+                // omregningAP_001
+                paragraph {
+                    text(
+                        Bokmal to "Derfor har vi beregnet pensjonen din på nytt.",
+                        Nynorsk to "Derfor har vi berekna pensjonen din på nytt.",
+                        English to "We have therefore recalculated your pension."
+                    )
+                }
+            }
+
+            showIf(
+                aarsakUtvandret and
+                        pesysData.ytelseskomponentInformasjon_safe.beloepEndring_safe.notEqualTo(BeloepEndring.UENDRET) and
+                        pesysData.inngangOgEksportVurdering.eksportForbudKode.isNull() and
+                        not(
+                            pesysData.opphoersbegrunnelseVedVirk_safe.begrunnelseBT_safe.equalTo(
+                                BRUKER_FLYTTET_IKKE_AVT_LAND
+                            )
+                        ) and
+                        not(
+                            pesysData.opphoersbegrunnelseVedVirk_safe.begrunnelseET_safe.equalTo(
+                                BRUKER_FLYTTET_IKKE_AVT_LAND
+                            )
+                        )
+            ) {
+                showIf(
+                    pesysData.beregnetpensjonPerMaanedVedVirk.grunnnpensjon.greaterThan(0) and
+                            not(pesysData.alderspensjonVedVirk.garantipensjonInnvilget) and
+                            not(pesysData.alderspensjonVedVirk.pensjonstilleggInnvilget) and
+                            not(pesysData.alderspensjonVedVirk.minstenivaaIndividuellInnvilget) and
+                            not(pesysData.alderspensjonVedVirk.minstenivaaPensjonistParInnvilget)
+                ) {
+                    // omregningGP_001
+                    paragraph {
+                        text(
+                            Bokmal to "Derfor har vi beregnet grunnpensjonen din på nytt.",
+                            Nynorsk to "Derfor har vi berekna grunnpensjonen din på nytt.",
+                            English to "We have therefore recalculated your basic pension."
+                        )
+                    }
+                }
+
+                showIf(
+                    pesysData.ytelseskomponentInformasjon_safe.beloepEndring_safe.notEqualTo(BeloepEndring.UENDRET)
+                            and pesysData.beregnetpensjonPerMaanedVedVirk.grunnnpensjon.greaterThan(0)
+                ) {
+                    paragraph {
+                        text(
+                            Bokmal to "Derfor har vi beregnet grunnpensjonen",
```

Comment:
Vil ikke "Derfor har vi beregnet grunnpensjonen din på nytt." vises to ganger om du har ingen tillegg? Tror du kan slette forrige avsnitt, den har egentlig samme kriterie.

Reply:
Godt poeng.